### PR TITLE
Update PPP for myself

### PR DESCRIPTION
**Pull request by paullee18**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ Friendnancial is a **desktop application, optimized for use by financial advisor
...
 ## Short Glossary for non-technical Users.
-| Technical Terms | Meaning                                                                                                                                    | 
-|-----------------|--------------------------------------------------------------------------------------------------------------------------------------------|
-| **GUI**         | Graphical User Interface - A system of interactive visual components for computer software                                                 | 
-| **CLI**         | Command Line Interface - A text based user interface to run programs                                                                       | 
-| **Index**       | A number indicating the order of a person within the contact list, used in conjunction with commands                                       |
-| **Parameter**   | Refers to the information typed along with the commands. For example the command `add n/John Doe` means that the parameter is `n/John Doe` | 
+| Technical Terms | Meaning                                                                                   |
```
Comment (gabyang) [inline docs/UserGuide.md:14]: This won't fix the glossary table, I alr fixed it in my latest PR.

**Review by gabyang — COMMENTED** [review]
Can remove the changes to glossary table

File: docs/UserGuide.md (hunk 1 above)
Comment (paullee18) [reply docs/UserGuide.md]: should be fine now

**Review by gabyang — APPROVED** [review]
lgtm, can squash your merge to clean up your commits

**Pull request merged**